src/server.cpp

```diff
@@ -297,6 +297,8 @@ Server::Server(
 
 	m_liquid_transform_every = g_settings->getFloat("liquid_update");
 	m_max_chatmessage_length = g_settings->getU16("chat_message_max_size");
+	m_csm_flavour_limits = g_settings->getU64("csm_flavour_limits");
+	m_csm_noderange_limit = (u32) g_settings->getS32("csm_flavour_noderange_limit");
```

Comment:
I'd be glad if you could do so. That's inconsistent and typecasts with negative numbers aren't always a good idea.